Gemfile

```diff
@@ -36,6 +36,7 @@ group :secondary do
   gem 'asciidoctor', '>= 0.1.0'
   gem 'liquid'
   gem 'maruku'
+  gem 'pandoc-ruby'
```

Comment:
The Gemfile is only used for development. When you use the gem, no other markdown engines are loaded (and there's no other dependencies either).